server.js

```diff
@@ -1685,6 +1685,33 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Apple Store version
+camp.route(/^\/applestore\/v\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var bundleId = match[1];  // eg, `324684580`
+  var format = match[2];
+  var apiUrl = 'https://itunes.apple.com/lookup?id=' + bundleId;
+  var badgeData = getBadgeData('AppleStore', data);
```

Comment:
On second thought, it seems `itunes app store` would be more consistent with the existing badge formats (c.f. `chrome web store`).